source/Halibut.Tests/BadCertificatesTests.cs

```diff
@@ -167,7 +168,7 @@ public async Task FailWhenClientPresentsWrongCertificateToListeningService(Clien
 
                 countingService.GetCurrentValue().Should().Be(0, "With a bad certificate the request never should have been made");
 
-                serviceLoggers[serviceLoggers.Keys.First()].GetLogs().Should()
+                serviceLoggers[serviceLoggers.Keys.First(x => x != nameof(MessageSerializer))].GetLogs().Should()
```

Comment:
This test assumed the logs it was interested in were going to be first in the list